.travis.yml

```diff
@@ -17,7 +17,9 @@ script:
 
 after_success:
   - sha256sum koncept.pdf
-  - sshpass -e scp -o StrictHostKeyChecking=no koncept.pdf ${FTP_USER}@${FTP_SERVER}:${FTP_FOLDER}
+  - mkdir .ssh
+  - ssh-keyscan -H ssaakademin.fn42.se,185.97.32.47 >> ~/.ssh/known_hosts
+  - sshpass -e scp koncept.pdf ${FTP_USER}@${FTP_SERVER}:${FTP_FOLDER}koncept-latest.pdf
```

Comment:
Jag hade föredragit att inte kalla något för "latest" eller "senaste", annat än ett alias/symlink/länk.

Reply:
Jag håller med om att det inte är någon sense i att ladda upp en koncept-fil med de 7 första tecknena i git hashen OCH en fil som heter koncept-latest.pdf. Dock så tycker jag att det är bra att kunna ha en länk som alltid pekar till den senaste versionen av PDF:en. Det finns säkert någon bra lösning som jag missat.

Reply:
Då laddar folk ner den med det filnamnet och vips så är vi här:
– _Så ser det inte ut för mig. Vilken version på PDF-filen har du?_
– _Jag har den senaste._

Reply:
Uppenbarligen. Bäst att bara döpa den till `koncept.GITHASH.pdf` och göra en symlänk till koncept.pdf

Reply:
Njae, för länkar man koncept.pdf så är det så den nedladdade kommer heta. Däremot är det som så att version och git-hashen står på varje sida. Just nu står det 2.0.0-RC0 48fef1d i min PDF.

Reply:
Jag tror att `latest` grejen kan förvirra folk men om den senaste alltid heter `koncept.pdf` och formuläret som man skickar in problem man har med text i boken även instruerar användaren att ladda hem den senaste versionen och sedan notera ner version och hash i ett fält i formuläret som är obligatoriskt så löser man nog det problemet. Då har man dessutom en permalänk som alltid pekar på senaste versionen.

Reply:
Man kan ju också använda något annat ord, till exempel `koncept-autobuild.pdf`, `koncept.github.pdf` eller `koncept_utkast.pdf`.

Reply:
I en commit jag gjorde igår så döps filerna till `koncept.<versionsnummer>.pdf` där versionsnumret är i enlighet med [Semantic Versioning](http://semver.org). Ett exempel: `koncept.2.0.0-RC0+b0150.ba564b1.pdf`. Jag håller på att fixa så att filerna ordnas så att senaste versionen är längst upp.

```bash
sshpass -e scp \
  koncept.pdf ${FTP_USER}@${FTP_SERVER}:${FTP_FOLDER}koncept-pdf/\
  koncept.$(cat VERSION.txt)+b$(printf "%04d" "${TRAVIS_BUILD_NUMBER}").${TRAVIS_COMMIT:0:7}.pdf
```
https://github.com/SverigesSandareamatorer/SSA-Akademin/pull/182/files#diff-354f30a63fb0907d4ad57269548329e3R22

Reply:
Men hur länkas den nuvarande versionen från en websida då? Hur uppdateras den länken? De allra flesta vill bara ha koncept.pdf. Vi kan ha en folder med koncept-<versionsnummer>.pdf för vårt bruk.

Reply:
Eftersom vi har versionsnumret på *varje sida* så behöver vi nog inte ha det i filnamnet också för de som *bara* vill ha hem en kopia av KonCEPT. Vi skulle kunna göra så att byggservern också gör en symlänk till den senaste filen så att man alltid har en permalänk för direkt nedladdning.

Reply:
Jag har försökt påpeka det tidigare, vi vill ha en fast länk för senaste release. Det är helt ok att sätta versionsnummer om man också laddar upp en symbolisk länk som uppdaterats.
Vi har redan versionsnumret på alla sidor och på tryckort-sidan, för de flesta bruk räcker det.

Reply:
> vi vill ha en fast länk för senaste release

Håller med. Fixar så att bygget gör en symlink till senaste bygget.